tests/test_integration_tox4.py

```diff
@@ -421,3 +421,11 @@ def test_pass_env(projdir, pass_env):
     assert result.returncode == 0
     assert "\nassertme\n" in result.stdout
     assert "\nNone\n" not in result.stdout
+
+
+def test_report_installed(projdir):
+    env = {"TRAVIS": "true"}
```

Comment:
```suggestion
    # tox4 only reports installed when a CI is detected
    env = {"CI": "true"}
```

Reply:
I consider the CI variable more obvious. Travis is a dead CI system that new folks won't even know in couple of years.